File: Java/DPR.java
class DPR {
    private int id;
    private String nama;
    private String bidang;
    private String partai;

    public DPR(int id, String nama, String bidang, String partai) {
        this.id = id;
        this.nama = nama;
        this.bidang = bidang;
        this.partai = partai;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNama() {
        return nama;
    }

    public void setNama(String nama) {
        this.nama = nama;
    }

    public String getBidang() {
        return bidang;
    }

    public void setBidang(String bidang) {
        this.bidang = bidang;
    }

    public String getPartai() {
        return partai;
    }

    public void setPartai(String partai) {
        this.partai = partai;
    }
}

File: Py/DPR.py
class DPR:
    
    def __init__(self, id=0, nama="", bidang="", partai=""):
        self.id = id
        self.nama = nama
        self.bidang = bidang
        self.partai = partai

    def get_id(self):
        return self.id

    def set_id(self, id):
        self.id = id

    def get_nama(self):
        return self.nama

    def set_nama(self, nama):
        self.nama = nama

    def get_bidang(self):
        return self.bidang

    def set_bidang(self, bidang):
        self.bidang = bidang

    def get_partai(self):
        return self.partai

    def set_partai(self, partai):
        self.partai = partai

